seleniumJSEnv/src/main/scala/org/scalajs/jsenv/selenium/SeleniumJSEnv.scala

```diff
@@ -0,0 +1,36 @@
+package org.scalajs.jsenv.selenium
+
+import org.scalajs.core.tools.io.VirtualJSFile
+import org.scalajs.core.tools.jsdep.ResolvedJSDependency
+import org.scalajs.jsenv.{AsyncJSEnv, ComJSEnv}
+import org.scalajs.jsenv.{JSRunner, AsyncJSRunner, ComJSRunner}
+
+class SeleniumJSEnv private (browser: SeleniumBrowser, keepAlive: Boolean)
+    extends AsyncJSEnv with ComJSEnv {
+
+  def this() =
+    this(Firefox, keepAlive = false)
```

Comment:
I can't remember whether I said otherwise before, but anyway: the `browser` should be a mandatory parameter. It does not make sense to choose `Firefox` by default over anything else. Or does it?

So this constructor should be:
```scala
def this(browser: SeleniumBrowser) =
  this(browser, keepAlive = false)
```
and `withBrowser` should be removed.